addons/l10n_eg/data/fiscal_templates_data.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="account_fiscal_position_non_egypt" model="account.fiscal.position.template">
```

Comment:
Where is the Egypt fiscal position?  Because right now, Non-Egypt will be by default.  If we can do an Egypt fiscal position auto-apply with country_id Egypt with a lower sequence than 20, it should work.  (that fiscal position can stay empty)